templates/base/server/.rtag/logstore.ts.hbs

```diff
@@ -0,0 +1,40 @@
+import fs from "fs";
+import path from "path";
+
+export default class LogStore {
+  private storageDir: string;
+  private partitionFiles: Map<string, number>;
```

Comment:
what happens when the server is bounced? will these file descriptors be closed gracefully?

Reply:
yeah the os will reclaim file descriptors on process exit